fern/definition/medical/fhir.yml

```diff
@@ -63,6 +75,48 @@ service:
           response:
             body: $GetConsolidatedQueryStatusResponse.Example0
 
+    createPatientConsolidated:
+      display-name: Create Patient's Consolidated Data
+      docs: Create Patient's Consolidated Data with a FHIR Bundle.
+      method: PUT
+      path: /patient/{id}/consolidated
+      path-parameters:
+        id:
+          type: string
+          docs: The ID of the Patient
+      request:
+        body: ConsolidatedBundleUpload
+      response:
+        type: fhir.Bundle
+      examples:
+        - name: Example1
+          path-parameters:
+            id: "12345678"
+          request: $ConsolidatedBundleUpload.Example1
+          response:
+             body:
+                resourceType: "Bundle"
+                id: "12345"
+                type: "transaction-response"
+                entry:
+                  - resource: "" # <---------------------- adding this works
```

Comment:
fix coming for this from fern. resource shouldnt need to be empty here in order to build